### DISCUSSION TITLE
Use a different linker

### DISCUSSION THREAD

**Discussion by kevinheavey** [messense/maturin-action#186] [Q&A]
Is it possible to change the linker used? I'd like to try using mold on linux and lld on macOS

**Comment by messense** [comment]
Did you encountered any issue when using a different linker? If you just want to know how to change the linker please refer to the documentation of Rust/Cargo, otherwise please provide more context, thanks.

**Comment by kevinheavey** [comment]
I know how to use a different linker on my own machine. I can't see how to do it with maturin-action

**Comment by messense** [comment]
The configuration should be similar to your local machine. 

You just need to install `mold` before executing `maturin-action` on macOS and use `before-script-linux` option of `maturin-action` to install `mold` before executing `maturin build` in docker container.
